content/WebAuthn/Concepts/Authenticator_Management/Implementation_Guidance/Vulnerability_Remediation.adoc

```diff
@@ -0,0 +1,127 @@
+== Vulnerability remediation implementation guidance
+
+When your application allows for a wide variety of authenticators you may run into the issue that one of them is found to have some sort of a vulnerability. When a vulnerability is discovered it’s easy enough to block that authenticator from continuing to register, but what do you do if users have already begun to leverage that authenticator in your environment? The guidance given on this page will teach you how to identify if a vulnerable authenticator is being used in your environment, and some options for remediation. 
+
+=== User experience
+
+Figure 1 demonstrates a sample user experience for a user who has registered an authenticator that was later identified as having a vulnerability
+
+image::../images/vulnrem1.jpg[]
+**Figure 1**
+
+In this flow a user has registered using an authenticator. Some time later it is identified that the authenticator used has a vulnerability. In the background your security team should identify all of your application users who have registered using the vulnerable authenticator. Once a user is identified, they should be alerted with instructions on **what the issue is** and **how can they remediate the issue**. 
```

Comment:
Consider adding a note that remediation requires collection authenticator attestation.  Authenticators regsitered with an attestation of none or anonymous will not be available for remediation.